[{"name": "test.test_most_frequent_first.MostFrequentFirst.test_content", "status": "errored", "message": "too many indices for array: array is 1-dimensional, but 2 were indexed", "passed": false, "points": ["p03-03.1"], "backtrace": ["  File \"/usr/lib/python3.10/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.10/unittest/case.py\", line 591, in run\n    self._callTestMethod(testMethod)\n", "  File \"/usr/lib/python3.10/unittest/case.py\", line 549, in _callTestMethod\n    method()\n", "  File \"/home/sonya/git/github/m1t0z/mooc-data-analysis-with-python-2021/part03-e03_most_frequent_first/test/test_most_frequent_first.py\", line 44, in test_content\n    for x in result[:,c]:\n"]}, {"name": "test.test_most_frequent_first.MostFrequentFirst.test_same_rows", "status": "errored", "message": "not enough values to unpack (expected 2, got 1)", "passed": false, "points": ["p03-03.1"], "backtrace": ["  File \"/usr/lib/python3.10/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.10/unittest/case.py\", line 591, in run\n    self._callTestMethod(testMethod)\n", "  File \"/usr/lib/python3.10/unittest/case.py\", line 549, in _callTestMethod\n    method()\n", "  File \"/home/sonya/git/github/m1t0z/mooc-data-analysis-with-python-2021/part03-e03_most_frequent_first/test/test_most_frequent_first.py\", line 30, in test_same_rows\n    np.testing.assert_allclose(sort_rows(a), sort_rows(result),\n", "  File \"/home/sonya/git/github/m1t0z/mooc-data-analysis-with-python-2021/part03-e03_most_frequent_first/test/test_most_frequent_first.py\", line 17, in sort_rows\n    n,m = a.shape\n"]}]